src/app/components/CreateUserForm.jsx

```diff
@@ -2,29 +2,79 @@
 /* eslint-disable no-unused-vars */
 import React, { useState } from 'react'
 /* eslint-enable no-unused-vars */
-import Link from 'next/link'
+import axios from 'axios'
+import { useRouter } from 'next/navigation'
+
 function CreateUserForm() {
 	const [showPassword, setShowPassword] = useState(false)
+	const [passwordMatchError, setPasswordMatchError] = useState(false)
 
 	const togglePassword = () => {
 		setShowPassword(!showPassword)
 	}
+
+	const router = useRouter()
+
+	async function onSubmit(event) {
+		event.preventDefault()
+		const formData = new FormData(event.target)
+
+		if (validatePasswords(formData)) {
+			formData.delete('confirmPassword')
+
+			const jsonData = {
+				username: formData.get('username').toString(),
+				password: formData.get('password').toString(),
+				email: formData.get('email').toString()
+			}
+
+			axios
+				.post(
+					process.env.NEXT_PUBLIC_BASE_URL + '/shared/user/',
```

Comment:
Create user has wrong URL, so can't post. New URL is /shared/auth/login

Reply:
Buenas Javi, esto no es así. La url que tu me indicas es para loguearse, con lo que esto no sería un error.

Reply:
Good afternoon Javi, this is not the case. The url that you indicate is for logging in, so this would not be an error.